modules/programs/k9s.nix

```diff
@@ -140,31 +159,52 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
+  config = let
+    isPre0_29 = (builtins.compareVersions cfg.package.version "0.29.0") < 0;
```

Comment:
Maybe we should rather use [lib.versionAtLeast](https://noogle.dev/f/lib/versionAtLeast).